lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -1049,6 +1049,22 @@ llvm::Error Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return llvm::Error::success();
   }
 
+  if (typeName == "LengthsRangeFill") {
+    NodeValue lengths;
+    ASSIGN_VALUE_OR_RETURN_ERR(lengths, getNodeValueByName(op.input(0)));
+    RETURN_ERR_IF_NOT(lengths.dims().size() == 1,
+                      "lengths must be a 1D vector.");
+
+    unsigned_t maxOutputSize;
+    ASSIGN_VALUE_OR_RETURN_ERR(maxOutputSize,
+                               loadInt(dict.find("maxOutputSize")->second));
```

Comment:
dict.find("maxOutputSize")->second if maxOutputSize does not exist would likely be a seg fault.